src/lib.ts

```diff
@@ -81,12 +82,14 @@ import {
   property,
   set,
   SongBoom,
+  SourceTerminal,
   sum,
   uneffect,
 } from "libram";
 import { acquire } from "./acquire";
 import { globalOptions } from "./config";
 import { garboValue } from "./value";
+import { gregLikeFightCount } from "./embezzler";
```

Comment:
We do not want to do this. I'm not sure where the best place to put the bunny banish functions given this, but we don't want to import embezzler to lib